tests/opf/test_solver.py

```diff
@@ -119,19 +116,14 @@ def test_dc(self):
         self.assertEqual(solution["success"], 1)
         self.assert_solution_valid(solution)
 
-        self.assert_approx_equal(solution["f"], 5216.026607, tol=1e-1)
-        self.assert_approx_equal(solution["gen"][1]["Pg"], 134.377585, tol=1e-1)
-        self.assert_approx_equal(solution["branch"][2]["Pt"], -56.2622, tol=1e-1)
+        self.assert_approx_equal(solution["f"], 17479.89, tol=1e-1)
 
     def test_ac(self):
         solution = solve_opf(self.case, opftype="AC")
         self.assertEqual(solution["success"], 1)
         self.assert_solution_valid(solution)
 
-        self.assert_approx_equal(solution["f"], 5296.686204, tol=1e-1)
-        self.assert_approx_equal(solution["bus"][2]["Vm"], 1.08662, tol=1e-1)
-        self.assert_approx_equal(solution["gen"][1]["Qg"], 0.031844, tol=1e-1)
-        self.assert_approx_equal(solution["branch"][0]["Qf"], 12.9656, tol=1e-1)
+        self.assert_approx_equal(solution["f"], 17551.89, tol=1e-1)
 
     @unittest.skip("shaky")
```

Comment:
You've made a lot of changes to this test, but it is being skipped. Does it pass? Is it stable now?

Reply:
test_ac_branchswitching is still shaky, test_ac is fine.